src/contracts/core/AVSDirectory.sol

```diff
@@ -132,31 +351,157 @@ contract AVSDirectory is
      *
      */
 
+    /// @inheritdoc IAVSDirectory
+    function operatorSetsMemberOfAtIndex(address operator, uint256 index) external view returns (OperatorSet memory) {
+        return _decodeOperatorSet(_operatorSetsMemberOf[operator].at(index));
+    }
+
+    /// @inheritdoc IAVSDirectory
+    function operatorSetMemberAtIndex(OperatorSet memory operatorSet, uint256 index) external view returns (address) {
```

Comment:
The `AtIndex` APIs don't seem that useful. I'm curious what the expected use case is

Reply:
Removed in https://github.com/Layr-Labs/eigenlayer-contracts/pull/860

Reply:
(agreed, they were not useful)